content/en/docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -256,7 +256,7 @@ your backups using a well reviewed backup and encryption solution, and consider
 encryption where possible.
 
 Kubernetes supports [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/), a feature 
-introduced in 1.7, and beta since 1.13. This will encrypt `Secret` resources in etcd, preventing
+introduced in 1.7, v1 beta since 1.13, and v2 alpha since 1.25. This will encrypt `Secret` resources in etcd, preventing
```

Comment:
back to the type of supported resources, I wonder if something the like of "This will encrypt resources with a `data` field like `Secret` and `ConfigMap` in etcd,..."